### ISSUE TITLE
Watchdog integration.

### ISSUE THREAD

**Issue by Lavalulu** [BeamMW/cuda-miner#6]
bigromanovToday at 8:39 PM
@Hardman please open issue in github, we will do it

This was regarding a short question in which I coined the idea of a Watchdog integration into the cuda miner. Ideally this would mean that its a watchdog that could be utlized by rig monitoring software solutions such as Awesome Miner or HiveOS.

Thanks in advance.

**Comment by emkman** [comment]
Yes would be great to expose json-http api to report sol rate. Simple watchdog could poll this and restart miner if rate == 0 for more than x minutes.
